apps/chat/tests/test_topic_bot.py

```diff
@@ -25,17 +24,18 @@ def test_safety_response(is_safe_mock):
     experiment.safety_layers.add(layer)
 
     expected = "Sorry I can't help with that."
+    experiment.get_llm_service = lambda: build_fake_llm_service([expected], token_counts=[1])
+
     bot = TopicBot(session)
     with patch.object(TopicBot, "_get_safe_response", wraps=bot._get_safe_response) as mock_get_safe_response:
-        with mock_experiment_llm(experiment, responses=[expected]):
```

Comment:
No. The patch doesn't patch anymore. I found this interesting, since this test and the one below it is pretty much the same. I am not 100% sure why yet, but didn't want to spend too much time on figuring it out atm